spec/parity/environment_spec.rb

```diff
@@ -31,8 +31,26 @@
     expect(Kernel).to have_received(:system).with(heroku_backup)
   end
 
+  it "correctly connects to the right Heroku app when the $PWD's name does not match the app's name" do
```

Comment:
"correctly" and "right" feel redundant on the same line. How about dropping "correctly"?